lib/modules/platform/gitea/index.ts

```diff
@@ -257,7 +257,7 @@ const platform: Platform = {
   ): Promise<any> {
     // TODO #22198
     const raw = (await platform.getRawFile(fileName, repoName, branchOrTag))!;
-    return JSON5.parse(raw);
+    return parseJson(raw, fileName);
```

Comment:
```suggestion
    return raw ? parseJson(raw, fileName) : null;
```
please add the null check